LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2016 Johannes Köster, the Rust-Bio team.
+Copyright (c) 2016 Johannes Köster, the Rust-Bio team, Google Inc.
```

Comment:
Hi, so how this works:
- yes, I'm an engineer at Google
- no, this is not a part of my daily work at Google, rather a fun side project for me. However, Google gets copyright to open source patches I submit when employed there.

Hope this works for you?